nvpair/src/lib.rs

```diff
@@ -372,6 +372,80 @@ impl NvEncode for str {
     }
 }
 
+impl NvEncode for &[u64] {
```

Comment:
Can we now change `impl NvEncode for [u64] {` to something like `(&self).insert_into(name, nv)`?